components/builder-db/src/error.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
```

Comment:
Excellent work with how the errors are being handled in this crate so far. We've come a long way :)